src/tiledb/cloud/client.py

```diff
@@ -107,6 +100,18 @@ def login(
             os.getenv("TILEDB_REST_IGNORE_SSL_VALIDATION", "False")
         )
 
+    if host is None:
+        host = config.default_host
+    # See sc-56351. Usually, a hostname doesn't include a protocol
+    # scheme, but our SDK strictly requires the http(s) scheme.
+    elif not host.startswith(("http://", "https://")):
+        if verify_ssl:
+            host = f"https://{host}"
+        else:
+            # Don't use https:// for non-SSL connections
+            # (defaulting to https:// when verify_ssl=False is confusing)
+            host = f"http://{host}"
```

Comment:
@ihnorton in light of @Shelnutt2's comment, perhaps we do this below? Assuming that our own review apps and staging redirect to HTTPS.

```suggestion
    elif not host.startswith(("http://", "https://")):
        host = f"http://{host}"
```